tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -45,6 +45,7 @@ const (
 	bookinfoYaml             = "samples/bookinfo/kube/bookinfo.yaml"
 	bookinfoRatingsv2Yaml    = "samples/bookinfo/kube/bookinfo-ratings-v2.yaml"
 	bookinfoDbYaml           = "samples/bookinfo/kube/bookinfo-db.yaml"
+	sleepYaml                = "samples/sleep/sleep.yaml"
```

Comment:
The major reason is that the egress case will need a pod with curl installed. To my knowledge pods from bookinfo do not have it. Besides check call metric does not have service label, it is hard tell where the cache hit is from. Restrict the call to only two services will make the verification easier: we can be sure the cache hit is from the destination service. There are several ways to achieve this, but since we need this sleep pod anyway for egress case, so I just reused it for intra mesh services case.